### PR TITLE
035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypt…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #1163
Script is common as PRs: #1310, #1311

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+function Test:TestStep_First_StartService()
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @istoimenova, Please clarify is this precondition or test?

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator"):Times(0)
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: RPC SetAudioStreamingIndicator"):Times(0)
+
+  commonTestCases:DelayedExp(10000)
+end
+
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @istoimenova, Please add commonFunctions:newTestCasesGroup("Test") where it is needed

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @dboltovskyi, updated as Precondition in 15bbd6556305449a954dde0b03d619414cd81597

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua]: @dboltovskyi, done in 15bbd6556305449a954dde0b03d619414cd81597

**Pull request merged**